wemulate/ext/settings/device.py

```diff
@@ -1,8 +1,7 @@
-from typing import List, Optional
-
 import netifaces
-
 import wemulate.core.database.utils as dbutils
+
+from typing import List, Optional
```

Comment:
Please add here a blank line to differentiate between internal and external dependencies to stick to our new import technique.

Reply:
Done